R/load_files.R

```diff
@@ -506,6 +507,7 @@ read_in_template_sheet_xlsx <- function(template_file, template_sheets, idx, pla
                    template_file[[idx]],
                    e))
     })
+    data.table::setDT(df)
```

Comment:
@bczech This one I do not understand.
Why it is needed here as in 496 line
```
df <-  df <- data.table::as.data.table(
        readxl::read_excel(
```
Maybe it should be?
```
 df <- data.table::serDT(
        readxl::read_excel(
````

Reply:
good catch

Reply:
✔️